app/components/Input.tsx

```diff
@@ -119,14 +119,17 @@ export type Props = React.HTMLAttributes<HTMLInputElement> & {
   onChange?: (
     ev: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => unknown;
+  innerRef?: React.RefObject<HTMLInputElement | HTMLTextAreaElement>;
```

Comment:
I'm having a bit of trouble with this typing, could use a set of eyes and suggestions